meet-addons-web-tutorial/src/app/page.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
```

Comment:
when would this page be used?

Reply:
Isn't the user always interacting with meet-addons-web-tutorial/src/app/meet/sidepanel/page.tsx?